packages/kbn-plugin-helpers/src/tasks/write_server_files.ts

```diff
@@ -37,16 +38,20 @@ export async function writeServerFiles({
 }: BuildContext) {
   log.info('copying source into the build and converting with babel');
 
+  const KIBANA_VERSION_79 = semver.satisfies(kibanaVersion, '~7.9.0');
+
   // copy source files and apply some babel transformations in the process
   await asyncPipeline(
     vfs.src(
       [
         'kibana.json',
-        ...(plugin.manifest.server
+        // always copy over the package.json file if we're building for 7.7-7.9
```

Comment:
`KIBANA_VERSION_79 ` is just checking that it's 7.9, should that matcher be updated?

Reply:
Comment is outdated